loader/src/main/kotlin/net/weavemc/loader/mixin/MixinSandbox.kt

```diff
@@ -180,21 +180,6 @@ class SandboxedMixinState(
     fun initialize() {
         if (initialized) return
 
-        injectService(
-            "net.weavemc.relocate.spongepowered.asm.service.IMixinService",
-            "net.weavemc.loader.mixin.SandboxedMixinService"
-        )
-
-        injectService(
-            "net.weavemc.relocate.spongepowered.asm.service.IMixinServiceBootstrap",
-            "net.weavemc.loader.mixin.DummyServiceBootstrap"
-        )
-
-        injectService(
-            "net.weavemc.relocate.spongepowered.asm.service.IGlobalPropertyService",
-            "net.weavemc.loader.mixin.DummyPropertyService"
-        )
-
```

Comment:
Fair enough, we used to not relocate mixin and therefore this was required.

Reply:
I'm reworking changes to the Mixin sandbox, that wasn't meant to be committed yet